### PR TITLE
Enrich examples to boost example quality KPI - Batch 3

### PR DESCRIPTION
**Pull request by SteveMutungi254**
Enrich examples to boost example quality KPI - Batch 3

### CODE REVIEW

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaDeletedGroup.md [hunk 1]
```diff
@@ -57,9 +57,9 @@ Get-EntraBetaDeletedGroup
 
 ## Description
 
-This cmdlet retrieves soft-deleted groups from a directory. When a group is deleted, it is soft deleted and can be recovered within 30 days. After 30 days, the group is permanently deleted and cannot be recovered.
+The `Get-EntraBetaDeletedGroup` cmdlet retrieves soft-deleted groups from the directory. Deleted groups can be recovered within 30 days, after which they are permanently deleted.
 
-Please note that soft delete currently applies only to Unified Groups (also known as Office 365 Groups).
+Note: Soft delete currently applies only to Unified Groups (Office 365 Groups).
```
Comment (msewaweru) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaDeletedGroup.md:62]: A note should go into an alert box.
```suggestion
Soft delete currently applies only to Unified Groups (Office 365 Groups).
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroupOwner.md [hunk 2]
```diff
@@ -50,24 +50,26 @@ In delegated scenarios, the signed-in user must have a supported Microsoft Entra
...
-cccccccc-2222-3333-4444-dddddddddddd
+id                                   displayName       @odata.type
+--                                   -----------       -----------
+aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb Adele Vance #microsoft.graph.user
 ```
 
 This example demonstrates how to retrieve the owner of a specific group.
 
-- `-GroupId` parameter specifies the ID of a group.
+- `-GroupId` Parameter specifies the ID of a group.
```
Comment (msewaweru) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroupOwner.md:65]:
```suggestion
- `-GroupId` specifies the ID of a group.
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroupMember.md [hunk 3]
```diff
@@ -111,25 +117,23 @@ This example retrieves all members within a group by group ID.
...
+Id                                   DeletedDateTime
+--                                   ---------------
+bbbbbbbb-7777-8888-9999-cccccccccccc
 ```
 
-This example retrieves the members of a specified group by its `GroupId` and selects only the `DisplayName` and `@odata.type` properties for each member.
+This example demonstrates how to retrieve group member by ID.
 
-- `-GroupId` specifies the ID of a group.
+- `-GroupId` Specifies the ID of a group.
```
Comment (msewaweru) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroupMember.md:136]:
```suggestion
- `-GroupId` specifies the ID of a group.
```

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Add-EntraGroupMember.md [hunk 4]
```diff
@@ -35,22 +35,29 @@ Add-EntraGroupMember
...
-Add-EntraGroupMember @params
+$group = Get-EntraGroup -Filter "DisplayName eq 'Contoso Marketing Group'"
+$user = Get-EntraUser -UserId 'SawyerM@contoso.com'
+Add-EntraGroupMember -GroupId $group.Id -RefObjectId $user.Id
 ```
 
 This example demonstrates how to add a member to a group.
 
+- `-GroupId` - specifies the unique identifier (Object ID) of the group to which you want to add a member.
+- -RefObjectId` - specifies the unique identifier (Object ID) of the member to be added to the group.
```
Comment (msewaweru) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Add-EntraGroupMember.md:59]:
```suggestion
- `-GroupId` - Specifies the unique identifier (Object ID) of the group to which you want to add a member.
- `-RefObjectId` - Specifies the unique identifier (Object ID) of the member to be added to the group.
```

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Add-EntraGroupOwner.md [hunk 5]
```diff
@@ -47,16 +50,14 @@ The `Add-EntraGroupOwner` cmdlet adds an owner to a Microsoft Entra ID group. Sp
...
-
-Add-EntraGroupOwner @params
+Add-EntraGroupOwner -GroupId $group.Id -RefObjectId $user.Id
 ```
 
 This example demonstrates how to add an owner to a group.
 
+- `-GroupId` - specifies the unique identifier (Object ID) of the group to which you want to add an owner.
+- `-RefObjectId` - specifies the unique identifier (Object ID) of the owner to be added to the group.
+
```
Comment (msewaweru) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Add-EntraGroupOwner.md:60]:
```suggestion
- `-GroupId` - Specifies the unique identifier (Object ID) of the group to which you want to add an owner.
- `-RefObjectId` - Specifies the unique identifier (Object ID) of the owner to be added to the group.

```

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraDeletedGroup.md [hunk 6]
```diff
@@ -67,9 +67,9 @@ Get-EntraBetaDeletedGroup
 
 ## Description
 
-This cmdlet retrieves soft-deleted groups from a directory. When a group is deleted, it is soft deleted and can be recovered within 30 days. After 30 days, the group is permanently deleted and cannot be recovered.
+The `Get-EntraDeletedGroup` cmdlet retrieves soft-deleted groups from the directory. Deleted groups can be recovered within 30 days, after which they are permanently deleted.
 
-Please note that soft delete currently applies only to Unified Groups (also known as Office 365 Groups).
+Note: Soft delete currently applies only to Unified Groups (Office 365 Groups).
```
Comment (msewaweru) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraDeletedGroup.md:72]:
```suggestion
Soft delete currently applies only to Unified Groups (Office 365 Groups).
```

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Remove-EntraGroupOwner.md [hunk 7]
```diff
@@ -41,12 +41,13 @@ The `Remove-EntraGroupOwner` cmdlet removes an owner from a group in Microsoft E
...
+$groupOwner = Get-EntraGroup -GroupId $group.Id | Get-EntraGroupOwner | Where-Object {$_.displayName -eq 'Adele Vance'}
+Remove-EntraGroupOwner -GroupId $group.Id -OwnerId $groupOwner.Id
 ```
 
 This example demonstrates how to remove an owner from a group in Microsoft Entra ID.
 
-GroupId - Specifies the ID of a group in Microsoft Entra ID.  
+- `GroupId` - Specifies the ID of a group in Microsoft Entra ID.  
 
 - `OwnerId` specifies the ID of an owner.
```
Comment (msewaweru) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Remove-EntraGroupOwner.md:52]:
```suggestion
- `OwnerId` - Specifies the ID of an owner.
```

**Review by msewaweru — APPROVED** [review]
Great job, @SteveMutungi254. I have left some minor suggestions.

**Review by KenitoInc — APPROVED** [review]

**Pull request merged**